src/ralph/domains/models/domains.py

```diff
@@ -127,6 +136,10 @@ class Domain(AdminAbsoluteUrlMixin, BaseObject):
         DNSProvider, blank=True, null=True,
         help_text=_("Provider which keeps domain's DNS")
     )
+    additional_services = MultiSelectField(
```

Comment:
what do you think about creating new model for this field and then using `models.ManyToMany` here? Take a look at business owner field here for example